internal/pkg/cli/task_run.go

```diff
@@ -26,37 +36,57 @@ var (
 	taskRunEnvPromptHelp = fmt.Sprintf("Task will be deployed to the selected environment. "+
 		"Select %s to run the task in your default VPC instead of any existing environment.", color.Emphasize(config.EnvNameNone))
 	taskRunGroupNamePromptHelp = "The group name of the task. Tasks with the same group name share the same set of resources, including CloudFormation stack, CloudWatch log group, task definition and ECR repository."
+
+	fmtTaskFamilyName = "copilot-%s"
+	fmtRepoName       = "copilot-%s"
+	fmtImageURL       = "%s:%s"
+)
+
+const (
+	defaultImageTag       = "copilot"
+	defaultDockerfilePath = "./Dockerfile"
+	clusterResourceType   = "AWS::ECS::Cluster"
 )
 
 type runTaskVars struct {
 	*GlobalOpts
-	count  int8
-	cpu    int16
-	memory int16
+	count  int64
+	cpu    int
+	memory int
 
 	groupName string
 
 	image          string
 	dockerfilePath string
+	imageTag       string
 
 	taskRole string
 
-	subnet         string
+	subnets        []string
 	securityGroups []string
 	env            string
 
-	envVars  map[string]string
-	commands []string
+	envVars map[string]string
+	command string
 }
 
 type runTaskOpts struct {
 	runTaskVars
 
 	// Interfaces to interact with dependencies.
-	fs     afero.Fs
-	store  store
-	parser dockerfileParser
-	sel    appEnvWithNoneSelector
+	fs    afero.Fs
```

Comment:
Is this used?

Reply:
It's used at Validate to determine whether the dockerfile path that user provides is valid